src/js/components/AppListComponent.jsx

```diff
@@ -126,17 +125,9 @@ var AppListComponent = React.createClass({
 
     if (props.filterTypes != null && props.filterTypes.length > 0) {
       appsSequence = appsSequence.filter(function (app) {
-        let appTypeIndex = 0;
-        if (app.container != null && app.container.type != null) {
-          appTypeIndex = AppTypes.indexOf(app.container.type);
-          if (appTypeIndex === -1) {
-            return false;
-          }
-        }
-
         /* Use .every for an INTERSECTION instead of UNION */
```

Comment:
I read it like this: If you want INTERSECTION instead of UNION you need to use ``.every`` instead of  ``.some``